sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/TinyintTests.java

```diff
@@ -316,10 +316,9 @@ public void testNumericRound() {
         );
     }
 
-    // Ignored because this fails in Postgres but here we get:
-    @Test @Ignore("Integer wrapping: https://github.com/feldera/feldera/issues/1186")
+    @Test
     public void testSelectOverflow() {
-        String error_message = "TINYINT out of range";
+        String error_message = "overflow";
```

Comment:
the other error message was a bit more informative.
but perhaps when we have source information this will compensate